models/Airplane/Airplane.jl

```diff
@@ -14,7 +14,18 @@
 # where ``(x, y, z)`` is the position of the C.G., ``(u, v, w)`` are the
 # components of velocity in ``(x, y, z)`` directions, ``(p, q, r)`` are body
 # rotation rates, and ``(ϕ, θ, ψ)`` are the Euler angles.
-#
+
+module Airplane  #jl
```

Comment:
also at the very top at the end of the first paragraph: it would help be nice to make reference to the picture with the diagram, see https://github.com/JuliaReach/NeuralNetworkAnalysis.jl/pull/198